Swiper.js

```diff
@@ -68,7 +68,8 @@ class Swiper extends Component {
     const { props, state } = this
     const propsChanged = (
       !isEqual(props.cards, nextProps.cards) ||
-      props.cardIndex !== nextProps.cardIndex
+      props.cardIndex !== nextProps.cardIndex ||
+      props.renderCard !== nextProps.renderCard
```

Comment:
@iyansr how does comparing the renderCard function help? I'm pretty sure it will not work or improve the swiper in any way, at least not without introducing a lot of other issues for other people.